core/src/main/java/org/elasticsearch/index/query/PrefixQueryParser.java

```diff
@@ -30,7 +30,9 @@
  */
 public class PrefixQueryParser implements QueryParser<PrefixQueryBuilder> {
 
-    private static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
+    public static final ParseField NAME_FIELD = new ParseField("_name").withAllDeprecated("query name is not supported in short version of prefix query");
+    public static final ParseField PREFIX_FIELD = new ParseField("value", "prefix");
```

Comment:
I'm not sure if this might be a case where both names are allowed, at least the doc state both version, not mentioning any deprecation.